### DISCUSSION TITLE
Quotes History

### DISCUSSION THREAD

**Discussion by anirban-oss** [monkeytypegame/monkeytype#4089] [Ideas]
I am using the website for a year now, I always miss not being able to go through the quotes I have typed in previous session, it would be great if we could see the history of different quotes we have gone through.

**Comment by o-x-e-y** [comment]
When the site was in its very early stages it used to have quote PBs but that got out of hand fast as the amount of quotes and users grew, but I've been thinking of bringing this back with localstorage which feels like it should be fairly easy. I assume adding a button in quote search that shows only quotes you typed/didn't type should be easy enough too, though I'm not super sure how useful that'd be (though some setting that doesn't repeat quotes would also be interesting?). I might look at this more this week or the next

**Reply by anirban-oss** [reply]
This is great news, I am also with you that local storage should be apt for this feature.
And not repeating recently typed quotes idea is also super cool!
I would love to have keyboard shortcuts like Shift + <-- arrow key to repeat quotes and other combinations for different operations on the typed words, let me know if i can help, i am trying to master python, ml, linux for now. i think i am good at design.

Monkeytype is my stress buster for any reason while debugging code. Thank you for the project. :)

**Comment by Jeff0502** [comment]
I think this sounds like an excellent idea. I would also like this feature, however, what about instead of listing all the quotes the user has typed in their lifetime, only the last (x) number can be shown. For example the last 5.
